pkg/pb/synthetic_monitoring/checks_extra.go

```diff
@@ -97,20 +100,16 @@ func (c *Check) Validate() error {
 
 	// timeout must be in [1, 10] seconds
 	if c.Timeout < 1*1000 || c.Timeout > 10*1000 {
-		return ErrInvalidCheckFrequency
+		return ErrInvalidCheckTimeout
```

Comment:
we should also validate that check frequency is grater then check timeout.

If we allow check frequency to be less then timeout, we would end up running concurrent checks for a target.

issue: https://github.com/grafana/synthetic-monitoring-api/issues/95

Reply:
thanks, I'll include that here